hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ECContainerReplicaCount.java

```diff
@@ -127,6 +143,21 @@ public Set<Integer> decommissioningIndexes() {
     return decommissionIndexes.keySet();
   }
 
+  /**
+   * Get a set containing all decommissioning only indexes, or an empty set if
+   * none are decommissioning.
+   * @return Set of indexes in decommission only.
+   */
+  public Set<Integer> decommissioningOnlyIndexes() {
```

Comment:
Please try to add a simply test for this, and if we change it to have an `includePendingAdd` flag, then test both scenarios. It may fit in as part of another test with decommission making it just one or two additional lines.

Reply:
Added tests